### ISSUE TITLE
Put Quirks/Modifiers into separate groups

### ISSUE THREAD

**Issue by Zesaric** [EmilyBjoerk/lsml#610]
I understand the idea behind putting Quirks and Modifiers into the same group but I think separating them would be an improvement.

Why? Quirks are the main pieces of info required to build a good loadout. I.e. it should be easy to see on one glance which type of weapon is encouraged by quirks.

Adding the modifiers from skills and modules bloats up this list, so right now the most important info (weapon quirk) is hidden in between all these values.

Personally, I do not need to see the modifiers from skills and modules at all because I know them. That's why I probably would collapse these groups if they were separate ones. Still, I understand why people might like to see them, so it makes sense to display them somehow.

To sum it up, the suggestion would be to have the following separate groups:
Quirks - Skill/Module Modifiers
or even
Quirks - Skill Modifiers - Module Modifiers

**Comment by Kmieciu4ever** [comment]
How about different text color for modifiers (green), structure and armor quirks (blue) and weapon quirks (red)?

**Comment by EmilyBjoerk** [comment]
I do not want to introduce another colour in the ui for this. The reason
being that it goes against information design theory to introduce more
different design elements. When all one really wants is a grouping.

**Comment by Kmieciu4ever** [comment]
I don't think we need another group. I would rather have an option to hide all efficiency modifiers just like I hide structure/armor quirks.

**Comment by EmilyBjoerk** [comment]
PGI has announced on MechCon 2016 that modules and pilot efficiencies (and quirks?) will be going the way of the dodo bird.

With that in mind I will down prioritise this a bit as there is a risk that I will have to redo everything anyway when those changes hit live. I will focus my time on other things for right now. I might still come back and do this if I have time over. For now, I'll put it to the future.

https://mwomercs.com/forums/topic/240718-december-roadmap-and-beyond/
https://www.youtube.com/watch?v=H6tgm_b2yWE

**Comment by Zesaric** [comment]
That's a good decision, I completely agree that this should be approached after the efficiencies revamp.

**Comment by sestrand** [comment]
PGI has done a lot of what Z and K thought up 6 years ago.  heh.  Colored groupings have made the 'in game' quirk system more readable.

**Comment by LiSongMWO** [comment]
I don't think that the game files contain a classification of the quirks, or at least I haven't stumbled over one. But even if that's the case, it's not particularly hard to implement in LSML. 

It's a matter of creating a function to assign a category based on the combination of "selector" and "specifier" attributes listed here: 

https://github.com/lisongmechlab/lsml/blob/2db92da3250e47e8a69dc0d1d4950870e2fda7ae/src/main/java/org/lisoft/lsml/model/modifiers/ModifierDescription.java#L52 

Some inspiration or potential reuse can be taken from:

https://github.com/lisongmechlab/lsml/blob/2db92da3250e47e8a69dc0d1d4950870e2fda7ae/src/main/java/org/lisoft/lsml/model/modifiers/AffectsWeaponPredicate.java#L36
